.github/workflows/archive-old-cards.yaml

```
@@ -0,0 +1,33 @@
+name: Update Activity Archive
+description: |
+  This action updates the activity archive so that cards finished in the last cycle
+  are moved to an "archive" column, and cards from two cycles ago are
+  "Archived" entirely (so they are not visible)
+on:
+  schedule:
+  # Run every 1st and 15th day of the month
+  - cron: "0 0 1,15 * *"
```

```suggestion
on:
  schedule:
  # Run every 1st and 15th day of the month
  - cron: "0 0 1,15 * *"
  # Add button to manually run workflow
    workflow_dispatch: []
```

What do you think about adding a [manual trigger](https://docs.github.com/en/actions/reference/events-that-trigger-workflows#workflow_dispatch) so we can also run this on the fly? In case a meeting is rescheduled or something?

---

Oh that's a cool idea! Actually what do you think if we took this one step further and:

1. Set this action to *only* be triggered by a manual click
2. Removed all the "check if two weeks old etc" logic from the action and just made it 1) Remove the cards from "Archive" and 2) Move "Done" to Archive
3. Added a step to our team deliverables sync meeting to "click the button to run this workflow"

---

I'm not sure about point 2. I can't decide if the logic helps or hinders the person running the workflow. But otherwise, yes :D